src/encoded/commands/create_mapping_on_deploy.py

```diff
@@ -123,6 +123,85 @@
     'Page',
 ]
 
+WEBPROD = 'fourfront-webprod'
```

Comment:
I might name these `ENV_xxx` so that variable name completion on `ENV_` will show the full list.

Reply:
Done